src/strings/en-us.json

```diff
@@ -339,10 +333,6 @@
     "EnablePhotosHelp": "Photos will be detected and displayed alongside other media files.",
     "EnableStreamLooping": "Auto-loop live streams",
     "EnableStreamLoopingHelp": "Enable this if live streams only contain a few seconds of data and need to be continuously requested. Enabling this when not needed may cause problems.",
-    "EnableThemeSongs": "Enable theme songs",
-    "EnableThemeSongsHelp": "If enabled, theme songs will be played in the background while browsing the library.",
-    "EnableThemeVideos": "Enable theme videos",
-    "EnableThemeVideosHelp": "If enabled, theme videos will be played in the background while browsing the library.",
```

Comment:
All four of these are still referenced in the code

Reply:
I thought it wasn't working, what should I do anyway?